exasol_script_languages_developer_sandbox/lib/vm_bucket/vm_slc_bucket.py

```diff
@@ -1,35 +1,51 @@
 from exasol_script_languages_developer_sandbox.lib.aws_access.aws_access import AwsAccess
 from exasol_script_languages_developer_sandbox.lib.logging import get_status_logger, LogType
 from exasol_script_languages_developer_sandbox.lib.render_template import render_template
+from enum import Enum
 
 STACK_NAME = "DEVELOPER-SANDBOX-VM-SLC-Bucket"
-BUCKET_NAME = "VMSLCBucket"
-ROLE_NAME = "VMImportRole"
+
+
+class OutputKey(Enum):
+    VMBucketId = "VMBucketId"
+    VMExportRoleId = "VMExportRoleId"
+    CfDistributionId = "CfDistributionId"
+    CfDistributionDomainName = "CfDistributionDomainName"
+
 
 LOG = get_status_logger(LogType.VM_BUCKET)
 
 
+def create_vm_bucket_cf_template() -> str:
+    # Simply map the output key enums values to them self and pass them to jinja.
+    # Thus, we ensure that the output keys in the cloudformation match with the values in class OutputKey
```

Comment:
Maybe, add to tbe explanation, that you parameterized the Cloudformation output keys in the templates

Reply:
Done